src/blocks/time.rs

```diff
@@ -104,11 +112,3 @@ pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
         }
     }
 }
-
```

Comment:
@MaxVerevkin we need this to be able to get the timezone if the user change `/etc/localtime`, right?

Reply:
Yes. @virajbhartiya added `use libc::tzset`, but I can't find it in libc docs?